src/strategies/safe-vested/index.ts

```diff
@@ -6,7 +6,7 @@ import { Multicaller } from '../../utils';
 export const author = 'dasanra';
 export const version = '0.1.0';
```

Comment:
```suggestion
export const version = '0.1.1';
```

Reply:
Thank you for suggesting this. I will set minor bump as I'm adding a new parameter and we try to follow SemVer.

I think the current active proposals are in a test space but anyway, it shouldn't break if the parameter is not configured 👍